et_replay/comm/backend/pytorch_dist_backend.py

```diff
@@ -996,18 +998,12 @@ def __init__(self, bootstrap_info, commsParams):
             if isinstance(self.commsParams, dict)
             else self.commsParams.backend
         )
-        # Import ucc plugin
-        if backend == "ucc":
-            # try OSS/setup.py
+        # Import Fairring
+        if backend == "fairring":
```

Comment:
Looks like "fairring" is the supported backend. Shall we add "fairring" into supportedC10dBackends in base_backend.py? and remove "ucc"?

Reply:
I think so. Will try to update. Thanks!
@Sergei-Lebedev